megalinter/utils.py

```diff
@@ -14,6 +14,8 @@
 from megalinter import config
 from megalinter.constants import DEFAULT_DOCKER_WORKSPACE_DIR
 
+SIZE_MAX_SOURCEFILEHEADER = 1024
```

Comment:
Or we can just make it real words that aren't spelling mistakes, like:

```suggestion
SIZE_MAX_SOURCE_FILE_HEADER = 1024
```

Reply:
The underscore is used here to subcategorize a name into relevant concepts. Size max of source file headers are three  concepts, for example, source file body could have been a concept (`SOURCEFILE_HEADER` along with `SOURCEFILE_BODY`), but isn't in this module (context).

Reply:
The underscores are helpful to the reader as delimiters of words since spaces aren't permitted.